src/AdapterPluginManager.php

```diff
@@ -45,7 +58,7 @@ class AdapterPluginManager extends AbstractPluginManager
      * @return void
      * @throws Exception\RuntimeException if invalid
      */
-    public function validatePlugin($plugin)
+    public function validate($plugin)
```

Comment:
Also, it ensures we're throwing container-interop exceptions, which we *should* be doing from plugin managers and factories and whatnot at this point; makes it easier for implementations to catch such exceptions and report them.